Sources/SwiftProtobuf/BinaryEncodingVisitor.swift

```diff
@@ -18,7 +18,7 @@ import Foundation
 /// Visitor that encodes a message graph in the protobuf binary wire format.
 internal struct BinaryEncodingVisitor: Visitor {
 
-  private var encoder: BinaryEncoder
+  var encoder: BinaryEncoder
```

Comment:
Does this need to be promoted to internal when the only change in this PR is within this type/file?